code/lib/manager-api/src/modules/refs.ts

```diff
@@ -168,12 +168,12 @@ export const init: ModuleFn<SubAPI, SubState> = (
 
       return Object.values(refs).find(({ url }: any) => url.match(source));
     },
-    changeRefVersion: (id, url) => {
+    changeRefVersion: async (id, url) => {
       const { versions, title } = api.getRefs()[id];
       const ref: API_SetRefData = { id, url, versions, title, index: {}, expanded: true };
 
       api.setRef(id, { ...ref, type: 'unknown' }, false);
```

Comment:
based on line 279 below this should be awaited right?
```suggestion
      await api.setRef(id, { ...ref, type: 'unknown' }, false);
```